src/GitHub.App/ViewModels/Dialog/Clone/RepositoryCloneViewModel.cs

```diff
@@ -118,6 +118,11 @@ public async Task InitializeAsync(IConnection connection)
                 SelectedTabIndex = 1;
             }
 
+            if (connection == null)
+            {
+                SelectedTabIndex = 2;
```

Comment:
When I open the clone dialog from the Start Page it always opens up to the URL tab. Is this why?

Reply:
When a GitHub or GitHub Enterprise account isn't explicitly targeted, it will default to using the URL tab.

When using this functionality I've found that more often than not, I'm cloning a repository that I've found on github.com rather than one of my own (or my organization's) repositories. Having to wait while a list of repositories that probably won't included the one I'm interested in isn't very useful (in fact it can be off-putting).

I think it makes more sense for the user to either explicitly target a GitHub/Enterprise account or paste the URL they were just looking at on github.com and Clone.

Maybe @donokuda has some thoughts?

Reply:
I get what you're saying about waiting to wait for a long list of repositories to load and being able to easily paste the URL, but I would expect the dialog on the Start to open to the first tab just like it does from Team Explorer.

Reply:
Yes we should definitely open the first tab rather than the URL tab. @jcansdale the reason it's slow is that the `github` org contains 2,461 repositories. Most users won't be members of an org with that many repositories and their list should load in 1 or 2 requests.